cmd/veneur/main.go

```diff
@@ -70,6 +71,10 @@ func main() {
 		if err != nil {
 			logrus.WithError(err).Fatal("error migrating splunk config")
 		}
+		err = prometheus.MigrateConfig(&conf)
+		if err != nil {
+			logrus.WithError(err).Fatal("error migrating prometheus config")
+		}
```

Comment:
You might need to update this as well.

Reply:
👍  That's what I get for not running go build again.

Reply:
Update this in what way? @arnavdugar-stripe

Reply:
Ah, new version doesn't emit errors. My bad.

Reply:
@jcrpaquin-stripe In a previous commit, @truong-stripe changed the signature of `prometheus.MigrateConfig` to remove the returned error, since it is always `nil`.